app/src/main/java/org/oppia/app/player/state/listener/StateKeyboardButtonListener.kt

```diff
@@ -2,5 +2,5 @@ package org.oppia.app.player.state.listener
 
 /** This interface helps to know when a keyboard action done button has been clicked. */
 interface StateKeyboardButtonListener {
-  fun onEditorAction(actionCode: Int)
+  fun onEditorAction(actionCode: Int){}
```

Comment:
This shouldn't be necessary, please revert (if a test activity needs to override the method and do nothing, I suggest doing that rather than defaulting this listener). I don't typically expect single-method interfaces to have default implementations--that's usually more often used for interfaces with a bunch of methods that are optional for overriding.

Reply:
Removed